src/main/java/frc/robot/subsystems/ArmJointSystem.java

```diff
@@ -59,7 +56,7 @@ public double getPositionDegrees(){
     }
```

Comment:
YOU GUYS IGNORED THIS COMMENT, SO HERE IT IS AGAIN!

Additionally, regarding encoders:

Because the NEO-integrated (relative encoder) is more trustworthy for use in constant motion compared to the absolute encoder, we will use it by default after startup.

So, update the subsystem to use that encoder. Update angle measurement, PID, config and such. Remember to consider gear ratio here.

The absolute encoder will still be used. Use it in the ctor after configure. Query it once and update the relative encoder with that position.

This gives us the best of both world: performance of relative encoder and absolute positioning on startup of absolute encoder.